src/app/components/CommentsList.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Comment from '../components/Comment';
+
+export default class CommentList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comment: [],
+    };
+  }
+  render() {
+    if (!this.state.comment.length) {
+      return null;
+    }
+
+    const comment = this.state.comment.map(comment => {
```

Comment:
```suggestion
    const comment = this.state.comments.map(comment => {
```
и тут сразу становится понятнее)